app/jobs/order_record_qc.rb

```diff
@@ -2,7 +2,8 @@ class OrderRecordQc
   @queue = :order_qc
 
   def self.perform(record_num)
-    deleted = deleted?(record_num) 
+    deleted = deleted?(record_num)
```

Comment:
What about just adding a conditional to this block?
* check for existence in Problems db
* if present, validate and skip create
* if not present, create problem